### PR TITLE
Add even more abu ids

### PR DESCRIPTION
**Pull request by kodawah**
for something completely different!

### CODE REVIEW

File: data/ignore.yaml [hunk 1]
```diff
@@ -62,6 +62,7 @@ abugames:
   '1100983': Planar Chaos Theme Deck Set of Four
   '1101008': Planeshift Theme Deck Set of Four
   '1101016': Portal Second Age Theme Deck Set of Five
+  '1101043': Planeswalker 30-Card Promo Deck Set of Five
```
Comment (axxroytovu) [inline data/ignore.yaml:65]: This one feels like it could easily stay in as a real product. We have plenty of "set of 5" type products.

Comment (kodawah) [reply data/ignore.yaml:65]: i mean, all of the above are potential products, im just not sure how widespread these old ones are

Comment (axxroytovu) [reply data/ignore.yaml:65]: Yeah, I think for consistency we should include these. 

I can update that if you don't want it to be part of this PR.

Comment (kodawah) [reply data/ignore.yaml:65]: that would be wonderful - we should maybe add a command to the new product script to generate the boilerplate in the given file to create new products on the fly!

Comment (axxroytovu) [reply data/ignore.yaml:65]: It already does that. The `review.yaml` file is formatted so you can just copy/paste it into the appropriate set file.

Comment (kodawah) [reply data/ignore.yaml:65]: yeah im saying we should add a command that prompts which edition to put that chunk in automatically

basically we shouldn't leave the script and manually copy and paste

**Pull request merged**